website/content/docs/developers/topologies.md

```diff
@@ -5,6 +5,36 @@ title: Writing a Topology
 A topology specifies components like spouts and bolts, as well as the relation
 between components and proper configurations.
 
+### Install Heron APIs for development
+
+Before getting started writing a topology, you need to install Heron API and 
+import its library to your own topology project.
+
+* Go to the releases page for Heron and download the Heron API installation 
```

Comment:
@lucperkins can we reference https://github.com/twitter/heron via params.github.home?
https://github.com/twitter/heron/blob/master/website/config.yaml#L40